modules/vaos/spec/services/jwt_spec.rb

```diff
@@ -5,14 +5,58 @@
 describe VAOS::JWT do
   subject { VAOS::JWT.new(user) }
 
-  let(:user) { build(:user, :mhv) }
+  let(:user) { build(:user, :vaos) }
+  let(:rsa_private) { OpenSSL::PKey::RSA.generate(4096) }
+  let(:jwt_regex) { %r{^[A-Za-z0-9\-_=]+\.[A-Za-z0-9\-_=]+\.?[A-Za-z0-9\-_.+/=]*$} }
 
   describe '#token' do
-    it 'encodes a payload' do
-      rsa_private = OpenSSL::PKey::RSA.generate 4096
-      allow(File).to receive(:read).and_return(rsa_private)
-      decoded = JWT.decode(subject.token, rsa_private.public_key, true, algorithm: 'RS512').first
-      expect(decoded['firstName']).to eq(user.first_name)
+    before { allow(File).to receive(:read).and_return(rsa_private) }
+
+    it 'returns a JWT string' do
+      expect(subject.token).to be_a(String).and match(jwt_regex)
```

Comment:
What exactly are you validating here? For example - what would be an example of a non-JWT string?

Reply:
A string that doesn't match the jwt regex: "this string would not validate." It has spaces, it does not have 3 base64 url encoded parts.